lib/events.js

```diff
@@ -20,6 +20,17 @@ EventEmitter.prototype._maxListeners = undefined;
 // added to it. This is a useful default which helps finding memory leaks.
 EventEmitter.defaultMaxListeners = 10;
 
+function ensureEvents(emitter) {
+  if (!emitter._events ||
+      emitter._events === Object.getPrototypeOf(emitter)._events) {
```

Comment:
Good point.  I wonder though, what is the reason for not just using `Object.hasOwnProperty(emitter, '_events')`?